media/css/firefox/family-nav.less

```diff
@@ -4,6 +4,10 @@
 
 @import "../sandstone/lib.less";
 
+#tabzilla:before {
+    background-color: @textColorSecondary;
+}
```

Comment:
Is this selector necessary? Most (all?) pages using the family nav do not have an element with the id `tabzilla` - instead `fxfamilynav-tabzilla` is used.